config/locales/de.yml

```diff
@@ -420,6 +422,7 @@ de:
     registered: Registriert
     rejected: Abgelehnt
     reserved: Reserviert
+  submit_application: Bewerbung abschicken
```

Comment:
The reason why I stated this and the translation in the comment below explicitly was, the context. Ohterwhise it would have been "Bewerbung erfassen".